packages/lumberaxe/lumberaxe.gemspec

```
@@ -40,7 +40,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "parser", ">= 2.5", "!= 2.5.1.1"
   spec.add_development_dependency "pry", ">= 0.14.2"
   spec.add_development_dependency "pry-byebug", "3.10.1"
-  spec.add_development_dependency "rails", ">= 6.0.6.1", "< 7.1"
+  spec.add_development_dependency "rails", ">= 6.0.6.1", "<= 7.1.3"
```

```suggestion
  spec.add_development_dependency "rails", ">= 6.0.6.1", "<= 7.1"
```

Let's not be specific about patch versions.